inc/Engine/Optimization/RUCSS/AbstractAPIClient.php

```diff
@@ -81,6 +81,14 @@ private function handle_request( array $args, string $type = 'post' ) {
 
 		$args['method'] = strtoupper( $type );
 
+		/**
+		 * RUCSS SAAS request arguments.
+		 *
+		 * @param array $args request arguments.
+		 * @returns array
+		 */
+		$args = apply_filters( 'rocket_rucss_api_endpoint', $args );
```

Comment:
I think that's not necessary, on the website side you can use the https://developer.wordpress.org/reference/hooks/http_request_args/ filter to modify the args for this specific request

Reply:
@Tabrisrp I could but it would heavily couple the code from the other plugin with the RUCSS url.